.github/workflows/release-image-builds.yml

```diff
@@ -84,7 +84,6 @@ jobs:
       - run: ./tests/test_stashcache.sh "docker.pkg.github.com/$GITHUB_REPOSITORY/stash-cache:${{ matrix.yum_repo_prefix }}minefield-${{ needs.get-timestamp.outputs.dtag }}"
   push-images:
     name: Push ${{ matrix.image }}:${{ matrix.yum_repo_prefix }}fresh image
-    if: github.event_name == 'push'
```

Comment:
I forget what this `github.event_name` corresponds to - is it the same as the `on:` up top?  If so, will this change the behavior for `repository_dispatch` events?  (by removing this restriction that it must be a `push`)

Reply:
Correct and that's the behavior that we want! It's something that I overlooked when I added repo dispatch support to this repo